hewr/R/to_epiweekly_quantile_table.R

```diff
@@ -30,7 +40,7 @@ to_epiweekly_quantile_table <- function(model_batch_dir) {
     report_epiyear,
     day_of_week = 7
   )
-  get_location_table <- function(model_run_dir) {
+  get_location_table <- function(model_run_dir, last_training_date) {
```

Comment:
Since this is being defined within `to_epiweekly_quantile_table()`, why not use that fact to have it not need `last_training_date` as an argument. Also see above RE: whether we might want to convert from last training date to first target date for clarity. 

```suggestion
  get_location_table <- function(model_run_dir) {
```

Reply:
https://github.com/CDCgov/pyrenew-hew/pull/287/commits/078b779ce0ae8c7f8bacbdef5953e3acbcafe99b